wynnset/tapestry-wp
# Deleting Child Nodes Causes Node to Become Uneditable

Closes #516 
Steps to test:
- Add a node that is a TYDE module
- Add some children
- Change the ordering of the children
- Delete one of the children
- Try to edit the parent node now, make sure you can
